src/jlib.erl

```diff
@@ -144,8 +145,17 @@ make_error_reply(#xmlel{name = Name, attrs = Attrs,
     NewAttrs = make_error_reply_attrs(Attrs),
     #xmlel{name = Name, attrs = NewAttrs, children = SubTags ++ [Error]};
 make_error_reply(Acc, Error) ->
-    make_error_reply(mongoose_acc:get(element, Acc), Error).
+    make_error_reply(Acc, mongoose_acc:get(element, Acc), Error).
 
+make_error_reply(Acc, Packet, Error) ->
+    case mongoose_acc:get(return_type, Acc, undefined) of
+        error_reply ->
+            ?CRITICAL_MSG("event=error_routing_loop,stanza=~p", [Packet]),
```

Comment:
It's definitely not critical IMO. I think it's rather debug. Plus, in `jlib` context it doesn't have to mean routing loop problem.

Reply:
I agree with the latter, but disagree with the former. If we try to generate an error response to an error response it means there is a bug in the code, and we have to resort to an emergency measure to prevent vm from crashing. Sounds rather serious to me.

Reply:
OK, I understand. Maybe ERROR then? For me, CRITICAL is a node-wide malfunction.